codec/legacy/codec.go

```diff
@@ -0,0 +1,31 @@
+package legacy
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
```

Comment:
naming! camel case

Reply:
complete codec directory is not needed was using it for some experimental changes would delete it.